apps/product-console/.env

```
@@ -0,0 +1,66 @@
+CRM_BASE_PATH='http://crm-plugin:4003/v1/'
+
+# Plugin Fees Configuration
+PLUGIN_FEES_HOST='plugin-fees'
+PLUGIN_FEES_PORT=4002
+PLUGIN_FEES_BASE_PATH='http://${PLUGIN_FEES_HOST}:${PLUGIN_FEES_PORT}/v1'
+
+# Reporter Configuration
+REPORTER_BASE_PATH='http://reporter:3002/v1'
+
+# Matcher Configuration
+MATCHER_BASE_PATH='http://matcher:8080/api/v1'
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)